Stateful Services/types/com/vantiq/engines/EngineMonitor_SpeedEvent_EventHandler/PartitionedAppState.json

```
@@ -9,26 +9,12 @@
       "type" : "Object",
       "unique" : false
     },
-    "EngineMonitor_SpeedEvent_EventHandler_DetectMalfunction_UpdateTempAndSpeedState" : {
-      "encrypted" : false,
-      "multi" : false,
-      "required" : false,
-      "type" : "Object",
-      "unique" : false
-    },
     "EngineMonitor_TempEvent_EventHandler_DetectMalfunction_UpdateStateState" : {
       "encrypted" : false,
       "multi" : false,
       "required" : false,
       "type" : "Object",
```

can you also update the types of the 2 remaining properties to be Map ?
(only for 1.34, not for maint-33)
```suggestion
      "type" : "Map",
```